components/uuf-renderablecreator-hbs/src/main/java/org/wso2/carbon/uuf/renderablecreator/hbs/helpers/registry/RuntimeHelperRegistry.java

```diff
@@ -199,7 +199,7 @@ public HelperRegistry registerHelpers(final String filename, final String source
      *
      * @param registry The handlebars instance.
      */
-    private void registerDefaultHelpers(final HelperRegistry registry) {
+    protected void registerDefaultHelpers(final HelperRegistry registry) {
         registry.registerHelper(WithHelper.NAME, WithHelper.INSTANCE);
         registry.registerHelper(IfHelper.NAME, IfHelper.INSTANCE);
         registry.registerHelper(UnlessHelper.NAME, UnlessHelper.INSTANCE);
```

Comment:
Remove the decorator registered in line [229](https://github.com/wso2/carbon-uuf/pull/169/files#diff-a16904c2242859bab7486d19974bcc43R229)